CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKHitTester.java

```diff
@@ -52,17 +110,158 @@ public CVKHitTester(CVKVisualProcessor parent) {
+            
+        } else {
+            // This is the resize path, image count is unchanged.
+        }
+        
+        swapChainResourcesDirty = false;
+        swapChainImageCountChanged = false;
+        
+        return ret;
+    } 
+    
+    
+    // ========================> Image <======================== \\
+    
+    private int CreateImages() {
+        CVKAssert(cvkSwapChain != null);
+        CVKAssert(cvkSwapChain.GetDepthFormat() != VK_FORMAT_UNDEFINED);
+        
+        int ret = VK_SUCCESS;
+   
+        try (MemoryStack stack = stackPush()) {
+            
+            int textureWidth = cvkSwapChain.GetWidth();
+            int textureHeight = cvkSwapChain.GetHeight();
+            int requiredLayers = 1;
+                        
+            // Create destination color image            
+            cvkImage = CVKImage.Create(cvkDevice, 
+                                            textureWidth, 
+                                            textureHeight, 
+                                            requiredLayers, 
+                                            //colorFormat, // Format TODO Not sure what the format should be - look at GL version
+                                            cvkSwapChain.GetColorFormat(),
+                                            VK_IMAGE_VIEW_TYPE_2D,
+                                            VK_IMAGE_TILING_LINEAR, // Tiling
+                                            VK_IMAGE_USAGE_COLOR_ATTACHMENT_BIT | VK_IMAGE_USAGE_TRANSFER_SRC_BIT, // TODO Usage 
+                                            VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT , // TODO - properties?
+                                            VK_IMAGE_ASPECT_COLOR_BIT,
+                                            "CVKImage cvkImage");  // TODO - aspect mask
+            
+            if (cvkImage == null) {
+                return 1;
+            }
+            
+            // Create depth image 
+            cvkDepthImage = CVKImage.Create(cvkDevice, 
+                                            textureWidth, 
+                                            textureHeight, 
+                                            requiredLayers, 
+                                            cvkSwapChain.GetDepthFormat(), // Format TODO Not sure what the format should be - look at GL version
+                                            VK_IMAGE_VIEW_TYPE_2D,
+                                            VK_IMAGE_TILING_OPTIMAL, // Tiling or VK_IMAGE_TILING_OPTIMAL
+                                            VK_IMAGE_USAGE_DEPTH_STENCIL_ATTACHMENT_BIT, // TODO Usage 
+                                            VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT, // TODO - properties?
+                                            VK_IMAGE_ASPECT_DEPTH_BIT,
+                                            "CVKImage cvkDepthImage");  // TODO - aspect mask
+            if (cvkDepthImage == null) {
+                return 1;
```

Comment:
Some VK error right?

Reply:
Added a comment as DepthImage may not be needed anyway